docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,69 @@
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+Users of Netobserv CLI can pass command line arguments to enable features or pass configuration options to the eBPF agent and flowlogs-pipeline.
```

Comment:
```suggestion
You can use the The Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and flowlogs-pipeline.
```

Reply:
Can we add more of these details we have here: https://docs.openshift.com/container-platform/4.12/observability/network_observability/netobserv_cli/netobserv-cli-reference.html